### DISCUSSION TITLE
Call functions within Next.js SSG

### DISCUSSION THREAD

**Discussion by sorcererxw** [vercel/community#709] [Help]
I use Golang for the API and Nextjs for the frontend in one Vercel project, like https://github.com/sorcererxw/vercel-nextjs-go-grpc-template.

Sometimes I need to call the Go API in SSG (getStaticProps), I tried to run the Go API on localhost in CI, but there is no Go in CI environment.

Of course, I can call the online address directly, like xxx.vercel.app, but this is the previous deployment and may conflict.

Is there any way to call local functions in CI?

**Comment by amyegan** [comment]
Hi @sorcererxw. Can you share the error you're seeing? I don't want to make any assumptions about what is failing for you.
